manifests/params.pp

```diff
@@ -5,6 +5,8 @@
       $trusted_cert_dir  = '/usr/local/share/ca-certificates'
       $update_cmd        = 'update-ca-certificates'
       $cert_dir_group    = 'staff'
+      $cert_dir_mode     = '2665'
```

Comment:
On an out of the box Ubuntu server, the certificate directory mode is 0555 - should this match the others?

Reply:
You're right. Debian has '2665', Ubuntu has '0555'. I'll add more condition logic to reflect this difference.